Vagrantfile

```diff
@@ -32,46 +31,42 @@ Vagrant.configure("2") do |config|
     # add the vagrant account to the docker group
     # this way the vagrant account can run docker without sudo
     usermod -aG docker vagrant
-
     # install podman
     apt update
     apt install -y podman
     # configure default init for podman
     sed 's;# init_path = "/usr/libexec/podman/catatonit";init_path = "/usr/bin/tini";g' -i /etc/containers/containers.conf
-
     # install miniconda3
     wget -P /tmp https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
     bash /tmp/Miniconda3-latest-Linux-x86_64.sh -b -p /home/vagrant/miniconda3
     rm /tmp/Miniconda3-latest-Linux-x86_64.sh
     /home/vagrant/miniconda3/bin/conda init --system
     /home/vagrant/miniconda3/bin/conda update conda -y
-
     # create a conda environment for development
     /home/vagrant/miniconda3/bin/conda create -y -n dbs python=3.8
+    # install pytorch and dependencies
+    /home/vagrant/miniconda3/bin/conda install -y pytorch torchvision torchaudio cpuonly -c pytorch
```

Comment:
I think the pytorch, srw, and shadow install steps need `-n dbs` or they will install into the base environment.

Reply:
I agree with that. Alternatively, the conda env can be activated first, then you will not have to specify the env name.

Also, you may want to source `conda.sh` so that the conda-related env variables are set, and you could use just `conda` command without the need to specify the full path.

```bash
. /home/vagrant/miniconda3/etc/profile.d/conda.sh
```

Reply:
fixed, thank you!

Reply:
my preference in a Vagrantfile is `-n <conda env name>` since activating the conda environment could affect later steps, and I don't have to remember to deactivate it